src/Components/Products.js

```diff
@@ -1,36 +1,78 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 
-import {addCartItem} from '../redux/store'
+import { addCartItemThunk, createOrderThunk } from '../redux/store';
 
-class _Products extends Component{
-
-  render(){
+class _Products extends Component {
+  async create(e, product) {
```

Comment:
Eliot told us not to have async awaits in React Components. But since we use them in our thunks, it should work the same if we just delete them here